src/test/java/pages/PlaylistPage.java

```diff
@@ -0,0 +1,89 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class PlaylistPage extends BasePage{
+    @FindBy (css = ".alertify-logs.top.right")
+    WebElement notification;
+    @FindBy (css = ".del.btn-delete-playlist")
+    WebElement deletePlaylistBtn;
+    @FindBy (css = ".fa.fa-plus-circle.create")
+    WebElement plusBtn;
+    @FindBy (css ="#playlists > nav > ul > li:nth-child(1)" )
+    WebElement newPlaylist;
+    @FindBy (css = "#playlists> ul> li:nth-child(4)")
```

Comment:
newPlaylist locator defined too long, and use Absolute paths.
You can define the locator with some element-specific attributes
//Example: 
```
@FindBy (xpath="//*[text()='New Playlist']" )
WebElement newPlaylist;
```